src/components/bcros/OfficeAddress.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <BcrosAddressAccordion
+    :name="name"
+    :items="addressItems"
+  />
+</template>
+
+<script setup lang="ts">
+import { storeToRefs } from 'pinia'
+const t = useNuxtApp().$i18n.t
+const route = useRoute()
+const business = useBcrosBusiness()
+const { currentBusinessAddresses } = storeToRefs(business)
+
+defineProps({
+  name: { type: String, required: true }
+})
+
+const addressItems = computed(() => {
+  const items = [{
+    label: t('label.address.officeType.registered'),
+    defaultOpen: true, // To confirm: will the registered office address panel be expanded by default?
+    address: currentBusinessAddresses.value.registeredOffice,
```

Comment:
I don't think firms have a 'registeredOffice' so this should also be wrapped in an 'if' similar to recordsOffice I think